src/palace/manager/sqlalchemy/model/work.py

```diff
@@ -1707,6 +1710,13 @@ def _set_value(parent, key, target):
                     lc["medium"] = doc.presentation_edition.medium
                 lc["licensepool_id"] = license_pool.id
                 lc["quality"] = doc.quality
+                collection_settings = (
+                    license_pool.collection.integration_configuration.settings_dict
+                )
+                lc["lane_priority_level"] = collection_settings.get(
```

Comment:
I verified that the correct value is definitely coming in from the collection settings.